<launch>
    <node pkg="segmentation" type="segmentation" name="detector" output="screen">
    <rosparam command="load" file="$(find segmentation)/config/extrinsic.yaml" />
    <param name="map_path" type="string" value="/home/wqx/RM/ForExperiment/src/Point-LIO-master/PCD/RMUC2025_X+10.pcd"/> 
    <param name="freq" type="int" value="20"/> 
    <param name="distance_threshold" type="double" value="0.12"/> 
    <param name="leaf_size" type="double" value="0.02"/>
    <param name="/use_sim_time" value="false" />
    <param name="prior_map_pub_en" type="bool" value="true"/>
    <param name="use_livox_cloud" type="bool" value="false"/>
    <param name="diverge_threshold" type="double" value="0.3"/>
    </node>
</launch>